bin/configurer/utils/idam-add-role.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -eu
+
+name=${1}
+description=${2}
+
+docker run --rm --network ${DATABASE_NETWORK:-compose_default} postgres:11-alpine psql --host ${DATABASE_HOST:-ccd-shared-database} --username postgres --command "INSERT INTO role(name, display_name) VALUES('${name}', '${description}') ON CONFLICT DO NOTHING" idam
```

Comment:
Probably silly question, but don't we have some kind of API to do it? Going directly to the IDAM db looks a bit smelly. Is it something being changes as part of new IdAM solution as well?